src/main/reactjs/public/i18n/fi-FI.json

```diff
@@ -97,6 +100,41 @@
           "empty": "Tyhjennä Valinnat",
           "search": "Hae"
         }
+      },
+      "contactRequestForm": {
+        "title": "Yhteydenottopyynnön jättäminen",
+        "description": "Tällä yhteydenottolomakkeella voit jättää valitsemillesi kääntäjille yhteydenottopyynnön. Kirjoitathan viestiin mahdollisimman tarkan kuvauksen tarvittavasta käännöstyöstä.",
+        "buttons": {
+          "cancel": "Peruuta",
+          "previous": "Takaisin",
+          "next": "Seuraava",
+          "submit": "Lähetä"
+        },
+        "steps": {
+          "0": "Vahvista kääntäjät",
+          "1": "Täytä yhteystiedot",
+          "2": "Kirjoita viesti",
+          "3": "Esikatsele ja lähetä",
+          "4": "Valmis"
+        },
+        "chosenTranslatorsForLanguagePair": "Valitut kääntäjät kieliparista",
+        "firstName": "Etunimi",
+        "lastName": "Sukunimi",
+        "email": "Sähköpostiosoite",
+        "phoneNumber": "Puhelinnumero",
+        "contactInfo": "Yhteystiedot",
+        "message": "Viesti",
+        "writeMessageHereLabel": "Kirjoita viestisi tähän",
```

Comment:
voisi siirtää formin in alle eli forum { .... }

Reply:
siirsin `formLabels` alle